assets/src/components/d3/createHistogram.js

```diff
@@ -1,4 +1,4 @@
-import * as d3 from 'd3'
+import * as d3 from 'd3/dist/d3.min'
```

Comment:
I don't feel we import things from `dist` folder.  why are you doing this?

Reply:
None of the imports in the project we use `dist` in path while importing. This needs to be changed.

Reply:
As per my understanding we cannot have an import from `dist` folder. That mean something in the way we process this needs to change.

Reply:
Sounds good, I'll look into what could be changed